docs/maintenance.md

```diff
@@ -0,0 +1,49 @@
+# Maintenance mode of storage nodes
+
+## Overview
+
+Storage nodes go into maintenance mode (MM) while internal procedures are required
+to troubleshoot storage service issues. Data on MM nodes MAY be temporarily
+unavailable. Otherwise, such nodes do not differ from fully functional ones.
+Nodes go into MM on their own.
+
+## Reflection in the network map
+
+To globally notify the network about the start of maintenance procedures, node
+MAY set the corresponding state in the network map. The ability to set MM state
+is determined by the global network setting. If MM is disallowed at the moment,
+node will not be able to set this state. Setting change is not retroactive.
```

Comment:
I don't quite get the `Setting change`. I would like a more explicit sentence.

Reply:
Paraphrased, check pls.